faq/building.inc

```diff
@@ -1361,6 +1361,19 @@ networks</a>, except that you use [--with-lsf].  For example:
 shell$ ./configure --with-lsf=/path/to/lsf/installation
 </geshi>
 
+Note: There are some dependencies needed to build with LSF.
+
+1. Network Information Service Version 2, formerly referred to as YP.
+   This is typically found in libnsl, but could vary based on your OS.
+
+   - On most linux distros, Solaris, NEC, and Sony NEWSs: these are found in libnsl.
+   - On RHEL: libnsl, libnsl2 AND libnsl2-devel are required.
+   - On AIX: it should be in libbsd.
+   - On HP-UX: it should be in libBSD.
+   - On IRIX < 6: it should be in libsun (IRIX 6 and later it is in libc).
```

Comment:
Open MPI hasn't supported AIX, HP-UX, or IRIX for many years.  It's probably not worth listing them.